example-apps/todo-mvc-example/source/app.ts

```diff
@@ -1,14 +1,22 @@
-import {bootstrap, bind} from 'angular2/angular2';
+// Load Global Styles
+import 'todomvc-common/base.css';
+import 'todomvc-app-css/index.css';
+
+import {bootstrap} from 'angular2/platform/browser';
```

Comment:
`angular2/browser/bootstrap` Is what's used in the Angular 2.0 quickstart: https://angular.io/docs/ts/latest/quickstart.html

That said, `angular2/boostrap` also works.  I'll defer to your wisdom.